lib/drivers/medtronic600/NGPHistoryParser.js

```diff
@@ -490,7 +520,10 @@ class TempBasalCompleteEvent extends NGPHistoryEvent {
   }
 
   get duration() {
-    return this.eventData.readUInt16BE(0x12) * 60;
```

Comment:
WTH? The event size can change? Is there a comment we can put here to explain why this happens? Or is it just :man_shrugging: ?

Reply:
I don't know why this happens. I just noticed this in the CLP code today.